acloud/resource_cluster.go

```diff
@@ -175,3 +180,53 @@ func resourceClusterDelete(ctx context.Context, d *schema.ResourceData, m interf
 
 	return diags
 }
+
+func WaitUntilClusterIsRunning(ctx context.Context, m interface{}, org string, cluster acloudapi.Cluster) error {
+	return WaitUntilClusterHasStatus(ctx, m, org, cluster.EnvironmentSlug, cluster.Slug, "running", 10*time.Minute)
```

Comment:
```suggestion
	return WaitUntilClusterHasStatus(ctx, m, org, cluster.EnvironmentSlug, cluster.Slug, ClusterStateRunning, 10*time.Minute)
```

I'd config `running` to a `const` variable.

Do we want to make the timeout configurable through a variable on the cluster resource, with the default being `10*time.Minute`?

```tf
resource "acloud_cluster" "staging_cluster" {
...
  cluster_running_wait_seconds = 120
}
```